### PR TITLE
Added a vhs tape

### PR DESCRIPTION
**Pull request by KaviiSuri**
I've added the VHS tape to demo the basic usage.

### CODE REVIEW

**Comment by KaviiSuri** [comment]
@Morphclue I've added a VHS tape to show the usage. 
I can also add a Github CI Action to generate it on changes.

File: demo.tape [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#   Left[@<time>] [number]          Press the Left Arrow key
+#   Right[@<time>] [number]         Press the Right Arrow key
+#   Up[@<time>] [number]            Press the Up Arrow key
+#   Down[@<time>] [number]          Press the Down Arrow key
+#   Ctrl+<key>                      Press the Control key + <key> (e.g. Ctrl+C)
+#
+# Display:
+#   Hide                            Hide the subsequent commands from the output
+#   Show                            Show the subsequent commands in the output
+
```
Comment (Morphclue) [inline demo.tape:46]:
```suggestion
```
I think adding the documentation isn't necessary.

**Review by Morphclue — CHANGES_REQUESTED** [review]
Thank you for the PR!
It would be super nice if you could add an action to automatically update the `.gif`.
You can decide if you want to do this in this PR or a new one. Either one is fine for me.

The newest commit on master has the possibility to go back with the `b`-key.
It would be cool if you could update the `.gif`.
It also might be interesting to go back by one and search for a new card e.g. `Red-eyes`.

**Comment by KaviiSuri** [comment]
Sure, I'll get the action done in another PR!

It didn't have the back option before, will update it soon!

File: demo.tape (hunk 1 above)
Comment (KaviiSuri) [reply demo.tape:46]: Yup, it was just generated by default form `vhs`

**Review by Morphclue — APPROVED** [review]

**Pull request merged**